client/rpcserver/handlers.go

```diff
@@ -424,6 +426,22 @@ func handleLogout(s *RPCServer, _ *RawParams) *msgjson.ResponsePayload {
 	return createResponse(logoutRoute, &res, nil)
 }
 
+// handleOrderBook handles requests for orderbook.
+// *msgjson.ResponsePayload.Error is empty if successful.
+func handleOrderBook(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
+	form, err := parseOrderBookArgs(params)
+	if err != nil {
+		return usage(orderBookRoute, err)
+	}
+	book, err := s.core.Book(form.Host, form.Base, form.Quote)
+	if err != nil {
+		errMsg := fmt.Sprintf("unable to retrieve order book: %v", err)
+		resErr := msgjson.NewError(msgjson.RPCOrderBookError, errMsg)
+		return createResponse(orderBookRoute, nil, resErr)
+	}
+	return createResponse(orderBookRoute, &book, nil)
```

Comment:
Practically speaking, an option to return the top N orders for both sides would be helpful.